src/main/webapp/app/markdown-editor/markdown-editor.component.ts

```diff
@@ -145,6 +146,13 @@ export class MarkdownEditorComponent implements AfterViewInit {
     }
 
     ngAfterViewInit(): void {
+        // Commands may want to add custom completers - remove standard completers of the ace editor.
+        this.aceEditorContainer.getEditor().setOptions({
+            enableBasicAutocompletion: true,
+            enableLiveAutocompletion: true,
+        });
+        this.aceEditorContainer.getEditor().completers = [];
```

Comment:
This removes the basic (local) auto completion from ace that provides dictionary completion (e.g. be -> between). I remove them here as they don't really help us (and nobody is using them atm anyway).